mobile_back_apple/cpp/backend_coreml/coreml_settings.h

```diff
@@ -0,0 +1,68 @@
+/* Copyright 2022 The MLPerf Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+#ifndef COREML_SETTINGS_H
+#define COREML_SETTINGS_H
+
+#include <string>
+
+const std::string coreml_settings = R"SETTINGS(
+common_setting {
+  id: "num_threads"
+  name: "Number of threads"
+  value {
+    value: "4"
+    name: "4 threads"
+  }
+}
+
+benchmark_setting {
+  benchmark_id: "IC_tpu_float32"
```

Comment:
Inference always runs using datatype of the model.
Benchmark id is only how people reading the configs. It does nothing.

Reply:
`benchmark_id` is just a string used to match a model defined in a setting file with a benchmark defined in the task file, so the app knows which model to use for which benchmark.

The model precision will be inferred from the model file (at least for TFLite) and be used in the preprocessing step to cast the input data type to match with the model input precision.

So like Danil stated, the data type in the `benchmark_id` is only for human relevant.

Reply:
I understand that having float32 in the benchmark_id gives the precision of input model , however is this the same precision in which inference runs ?

Reply:
We should remove the data type in benchmark_id later. I added this by copy & paste these settings to make parsing logic of `flutter/cpp/binary/main.cc` happy so that we can move forward :-)

Reply:
Thanks for clarifying that @freedomtan